test/src/sdk-harness/test_projects/auth/mod.rs

```diff
@@ -740,16 +710,15 @@ async fn when_incorrect_predicate_address_passed() {
 async fn can_get_predicate_address_in_message() {
     // Setup predicate address.
     let hex_predicate_address: &str =
-        "0xeddbd8cc2be5fb1d4ae24a5f789e61308750ff698865f311c9818d7eb76d4777";
+        "0xf70528e17820e36c0868d25aac804f3e0750f90f0840d9e1d68a7e4e3c9290a8";
     let predicate_address =
         Address::from_str(hex_predicate_address).expect("failed to create Address from string");
-    let predicate_bech32_address = Bech32Address::from(predicate_address);
 
     // Setup message
     let message_amount = 1;
     let message = Message {
-        sender: Bech32Address::default(),
-        recipient: predicate_bech32_address.clone(),
+        sender: Address::default(),
+        recipient: predicate_address,
         nonce: 0.into(),
         amount: message_amount,
```

Comment:
we shuold probably update this to also be u128; i think where possible we should update amount to u128 since that is the default balance type it seems